clientv3/mirror/syncer.go

```diff
@@ -78,7 +95,7 @@ func (s *syncer) SyncBase(ctx context.Context) (<-chan clientv3.GetResponse, cha
 			// If len(s.prefix) != 0, we will sync key-value space with given prefix.
 			// We then range from the prefix to the next prefix if exists. Or we will
 			// range from the prefix to the end if the next prefix does not exists.
-			opts = append(opts, clientv3.WithPrefix())
+			opts = append(opts, clientv3.WithRange(prefixEnd(s.prefix)))
```

Comment:
It will change.  With `clientv3.WithPrefix()`, it means `op.end = getPrefix(op.key)`. Range end is based on the current key of the get request. 

For example, we want to get all kvs with prefix `a`. In the first batch, the range will be `[a, b)`.  we may get `[a, ab，abc]` . In the next batch, the key is `abc\0x00`. The range will be `[abc\0x00, abc\0x01)`。If there is any key in the range `[abc\0x11, b)`, like `abd`. It will be skipped.